src/eventkit.cs

```diff
@@ -683,17 +683,19 @@ interface EKVirtualConferenceDescriptor
 		string ConferenceDetails { get; }
 	}
 
+	delegate void VirtualConferenceHandler (EKVirtualConferenceDescriptor virtualConferenceDescriptor, NSError error);
```

Comment:
This is the reason I asked for it, there should be to diff ones, if you look at:
```csharp
void FetchAvailableRoomTypes (Action<NSArray<EKVirtualConferenceRoomTypeDescriptor>, NSError> completionHandler);
void FetchVirtualConference (string identifier, Action<EKVirtualConferenceDescriptor, NSError> completionHandler);
```

The parameters are diff, you need to create two diff ones. one for FetchAvailableRoomTypes with EKVirtualConferenceRoomTypeDescriptor as a first param and a second one for FetchVirtualConference with EKVirtualConferenceDescriptor as the first param. 
```
EKVirtualConferenceRoomTypeDescriptor != EKVirtualConferenceDescriptor
```

Reply:
Ah good catch, I completely missed that!